druid/src/widget/container.rs

```diff
@@ -77,6 +79,36 @@ impl<T: Data> Container<T> {
         self.background = None;
     }
 
+    /// Builder-style method for setting the foreground for this widget.
+    ///
+    /// This can be passed anything which can be represented by a [`BackgroundBrush`];
```

Comment:
I guess `BackgroundBrush` is no longer a great name. Not a change we need to worry about in this PR though.

Reply:
Yeah I was thinking the same since I was preparing the PR. Since this didn't make it in time should I prepare another PR for the `BackgroundBrush` change?

Reply:
If you wish, sure. After we merge this one here.

Reply:
Should I just rename it to `Brush`? I am also thinking `PrimitivesBrush`

Reply:
`Brush` sort-of clashes with Piet's [`Brush`](https://docs.rs/piet/latest/piet/trait.RenderContext.html#associatedtype.Brush) so perhaps not. Given that it is defined in `painter.rs` I'm also thinking maybe `PaintBrush`. Although we could maybe go with something non-brush even, given that it supports a full `Painter` widget as a variant. 🤔